fiddler/datadog_checks/fiddler/check.py

```diff
@@ -0,0 +1,152 @@
+# import datetime
+# import json
+# from requests.exceptions import Timeout
+
+# import yaml
+import time
+
+import fiddler as fdl
+
+from datadog_checks.base import AgentCheck
+
+# from typing import Any
+
+
+# We will get one hours worth data in chunks and process them.
+bin_size = 3600
+
+# metrics_list = ['accuracy', 'histogram_drift', 'output_average', 'feature_average', 'traffic_count']
+metrics_list = ['accuracy']
+tags = list()
+
+# Connection parameters for Fiddler. Currently hard coded to get some testing done. Will be parameterizing this later.
+URL = 'https://demo.trial.fiddler.ai'
+ORG_ID = 'demo'
+AUTH_TOKEN = 'K4ph7ORDcIO2xVIEA6KxL1o1zHjZockgurhCOZOUSVs'
+
+client = fdl.FiddlerApi(url=URL, org_id=ORG_ID, auth_token=AUTH_TOKEN)
+
+
+def create_tags(**tags_in):
+    tags.clear()
+    tags.append("project:" + tags_in['project'])
+    tags.append("model:" + tags_in['model'])
+    if len(tags_in) > 2:
+        tags.append("feature:" + tags_in['feature'])
```

Comment:
I'm not sure to understand the purpose of the `tags` global variable here. I think you do not really need it actually. What do you think of this approach?

```python
def create_tags(**kwargs):
    tags = []

    for key, value in kwargs.items():
        tags.append(f"{key}:{value}")

    return tags
```

or even 

```python
def create_tags(**kwargs):
    return [f"{key}:{value}" for key, value in kwargs.items()]
```

And then use the value returned?

Reply:
Good point. Made the change.